Sources/Sentry/include/SentryNetworkTracker.h

```diff
@@ -16,11 +16,14 @@ static NSString *const SENTRY_NETWORK_REQUEST_TRACKER_SPAN = @"SENTRY_NETWORK_RE
 
 - (nullable NSDictionary *)addTraceHeader:(nullable NSDictionary *)headers;
 
-- (void)enable;
+- (void)enableNetworkTracking;
+
+- (void)enableBreadcrumbs;
```

Comment:
`l`: To be consistent with the naming above. Same applies, of course, for the property isBreadcrumbEnabled.
```suggestion
- (void)enableNetworkBreadcrumbs;
```